docs/snippets/future-work-improved-metric.md

```diff
@@ -1 +1 @@
-`infrastructure` `community-engagement` `learning-experience`
+`infrastructure` `platform` `simulator` `telemetry` `environment` `community-engagement` `learning-experience` `accuracy` `pose` `numsteps` `speed` `noise` `learning` `multi-object` `generalization` `compositional` `deformations` `features-and-morphology` `scale` `real-world` `dynamic` `goal-policy` `abstract` `adversarial` `transfer` `biological-realism`
```

Comment:
Do we want to include `biological-realism`? I wonder whether it will give the wrong incentives?

Maybe for the one item where I saw it used (movement for voting), we can change the benefit to `unknown` or the like, as part of the task is determining the implications (we just have an intuition that it might be worthwhile)?

Reply:
yes I was debating about this as well. I'll remove it and just leave the improved metric empty for that one and add a note in the description